en/views.rst

```diff
@@ -460,7 +462,7 @@ Contacts plugin::
     {
         public function view_active()
         {
-            $this->layout = 'Contacts.contact';
+            $this->getView()->layout('Contacts.contact');
         }
```

Comment:
Actually changing `$viewClass` property after view instance is created does nothing. You would need to call createView() or call getView() passing view class as param for new object to be created.